src/container-manager/docker-files/lumino-explorer/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: "2.2"
+
+services:
+    mongo-db:
+      image: "mongo"
+      restart: "no"
+      networks: ["rsk-network"]
+      container_name: "mongo-db"
+      ports:
+        - "27017"
+    lumino-explorer:
+        #environment:
+          #EXPLORER_API_URL: "http://explorer-api:8080/api/v1/" # this can use a dns name defined on hosts file to map to the final ip.
+          #EXPLORER_PORT: 8080
+          #RSK_ENDPOINT: "http://rsk-node:4444"
+          #MONGO_DB_HOST: "mongo-db"
+          #TOKEN_NETWORK_REGISTRY: 0xed8c9163F888Bc2f9C4F299325003DA5fC8676DD
```

Comment:
Actually is there for a reason, to know which environment variables are available to use instead of going to the script to check them, but anyway i think i have to update them.